src/routes/AuthRoutes.js

```diff
@@ -1,9 +1,11 @@
 import express from 'express';
-import { register, login } from '../controllers/AuthController.js';
+import { register, login, forgotPassword } from '../controllers/AuthController.js';
 
 const router = express.Router();
 
 router.post('/auth/register', register);
```

Comment:
I was thinking, the authRoutes are currently look like this ` localhost:5000/api/auth/register` or `localhost:5000/api/auth/login`. 

Maybe no need `api` in the routes for the authRoutes?

Reply:
I thought all the routes for the back-end started from the api, didn't they?

After a quick search on ChatGPT, do you still want to remove api from the route?

![Screenshot 2023-12-08 at 9 56 26 pm](https://github.com/irene2mana-T3A2-2023/SakuraPantryServer/assets/98362650/43e48994-79db-4ae8-8d14-7a1a9ccd4810)

Reply:
all good, we def can keep the `api`.